src/components/CreditCardInformation.jsx

```diff
@@ -81,6 +82,15 @@ const CheckoutForm = ({ payableAmount, closePaymentModal }) => {
             )}
             {!paymentSuccessful && (
                 <>
+                    {user && user.plan !== 'Basic' && (
+                        <Alert status="warning" mb={4}>
+                            <AlertIcon />
+                            <AlertDescription>
+                                <Text>Your current plan is {user.plan}. </Text>
+                                <Text>Are you sure you want to pay again?</Text>
```

Comment:
What happens if a user pays again?

Reply:
they can just pay. I am trying to fix the workflow.